pkg/ingress/pod.go

```diff
@@ -64,9 +64,9 @@ func (c *podController) onAdd(obj interface{}) {
 	if !c.controller.namespaceWatching(key) {
 		return
 	}
-	log.Debugw("pod add event arrived",
-		zap.Any("object", obj),
-	)
+	//log.Debugw("pod add event arrived",
```

Comment:
We may still preserve this logging but don't show the object (just adding the name).

Reply:
Done, I print key (namespace + name) instead.